shell/components/CodeMirror.vue

```diff
@@ -24,18 +24,23 @@ export default {
     asTextArea: {
       type:    Boolean,
       default: false
-    }
+    },
+    showKeyMapBox: {
+      type:    Boolean,
+      default: false
+    },
   },
 
   data() {
     return {
-      codeMirrorRef: null,
-      loaded:        false
+      codeMirrorRef:       null,
+      loaded:              false,
+      showKeyMap:          this.showKeyMapBox,
```

Comment:
In this case we don't want to change `showKeyMapBox` in the parent component (it would return a console  error).
I didn't add this showKeyMapBox because it would be propagated to `<code-mirror>` component.